job/models.py

```diff
@@ -4548,6 +4548,12 @@ class REoptjlMessageOutputs(BaseModel, models.Model):
         default = list,
     )
 
+    has_stacktrace = models.BooleanField(
+        blank=True,
+        default=False,
+        help_text="True if the error message has a stacktrace, false otherwise"
```

Comment:
@rathod-b I think it could be helpful if you add a brief note as to why some error messages might have slack traces and some not

Reply:
Updated! Please take a look